pkg/shell/superuser.jsx

```diff
@@ -119,7 +126,33 @@ const UnlockDialog = ({ proxy, host }) => {
     let body = null;
     let footer = null;
 
-    if (prompt) {
+    if (bridges) {
+        const apply = () => {
+            setBridges(null);
+            start(bridge);
+        };
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17536-20220708-104722-284c800d-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)